src/autopas/tuning/utils/FeatureVector.h

```diff
@@ -44,10 +45,12 @@ class FeatureVector : public Configuration {
    * @param dataLayout
    * @param newton3
    * @param cellSizeFactor
+   * @param interactionType
    */
   FeatureVector(ContainerOption container, double cellSizeFactor, TraversalOption traversal,
-                LoadEstimatorOption loadEstimator, DataLayoutOption dataLayout, Newton3Option newton3)
-      : Configuration(container, cellSizeFactor, traversal, loadEstimator, dataLayout, newton3) {}
+                LoadEstimatorOption loadEstimator, DataLayoutOption dataLayout, Newton3Option newton3,
+                InteractionTypeOption interactionType)
+      : Configuration(container, cellSizeFactor, traversal, loadEstimator, dataLayout, newton3, interactionType) {}
```

Comment:
I don't quite get why `interactionType` is needed here. Can you add a brief comment?

Reply:
Because I added `interactionType` to the `Configuration` object which `FeatureVector` inherits` from.
Looking through the code again, I could probably remove this attribute within AutoPas. But I thought it would be nice to have a quick way to find out to what interaction type a configuration belongs to.
Let me know what you think about it